charts/sbombastic/values.yaml

```diff
@@ -36,6 +36,32 @@ storage:
     requests:
       cpu: 250m
       memory: 300Mi
+  postgres:
+    # Name of existing secret containing the URI to connect to external Postgres.
+    # Expects a `uri` key. The sslmode parameter will be ignored.
```

Comment:
I would add an example of how the `uri` should look like or suggest to read our docs (I haven't checked the docs PR, hopefully there's something that covers that in there)